src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -287,6 +287,14 @@ namespace AirflowNetworkBalanceManager {
         bool UpdateAirflowNetworkMyOneTimeFlag = true;
         bool UpdateAirflowNetworkMyOneTimeFlag1 = true;
 
+        // CalcAirflowNetworkAirBalance variables
+        int ErrCountVar = 0;
+        int ErrCountHighPre = 0;
+        int ErrCountLowPre = 0;
+        int ErrIndexHighPre = 0;
+        int ErrIndexVar = 0;
+        int ErrIndexLowPre = 0;
```

Comment:
These have to be reset in the clear_state method.

Reply:
Ok, Thanks for pointing this out!

Reply:
No problem!  And they are conflicted as I had addressed this in my static pass, so I'll just clean up the conflicts and we'll move on.